pages/updatePaymentMethod.js

```diff
@@ -43,278 +43,270 @@ const AlignedBullets = styled.ul`
   width: max-content;
 `;
 
-class UpdatePaymentPage extends React.Component {
```

Comment:
Same here let's keep things simple, no need to convert to functional components unless it brings a clear value

Reply:
Addressed in https://github.com/opencollective/opencollective-frontend/pull/6193 😄